CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing
+
+This file provides general guidance for developers that are creating or updating REST API definitions for Azure.
+
+## Table of Contents
+<!--
+  You should regenerate the TOC using the `markdown-toc` node package after making changes to this file.
+
+      npx markdown-toc -i --maxdepth 4 CONTRIBUTING.md
+  -->
+
+<!-- toc -->
```

Comment:
Just a reminder to regen toc before merge.

Reply:
Thanks for the reminder!  Done.